scss/tech.scss

```diff
@@ -0,0 +1,50 @@
+
+.techs-container{
+  display:flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+}
+
+.tech{
+  border: 1px solid $green;
+  border-radius: 4px;
+  flex-basis: 18%;
+  margin:20px;
+}
+
+.tech-content{
+  align-items:center;
+  background-color: $green;
+  border-radius: 4px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  flex-grow: 1;
+  height: 100%;
+  padding: 10px 30px;
+  margin: 10px -10px -10px 10px;
+  text-align: center;
+}
+
+.tech img{
+  height: $tech-logo-size;
+  width: $tech-logo-size;
+}
+
+.tech p{
+  font-family: $font;
+  font-size: $text-medium;
+}
+
+.tech p:nth-child(2){
+  margin-bottom:10px; 
+}
+
+.tech p:last-child{
+  font-size: $text-small; 
+}
+
+.tech p:nth-last-child(2){
```

Comment:
For this last classes, use a class in the element and style them via that class :smile: